minard/views.py

```diff
@@ -1296,6 +1318,33 @@ def pca_run_detail(run_number):
                            run_number=run_number,
                            run=run)
 
+@app.route('/burst_run_detail/<run_number><subrun><sub>')
+def burst_run_detail(run_number, subrun, sub):
+    server = couchdb.Server("https://snoplus:"+app.config["COUCHDB_PASSWORD"]+"@"+app.config["COUCHDB_HOSTNAME"])
```

Comment:
The part of this function which fetches the results should also be moved to the new file which deals with accessing the database.